protos/exocore/core/config.proto

```diff
@@ -33,6 +35,34 @@ message NodeCellConfig {
     }
 }
 
+message NodeStoreConfig {
+    EntityIndexConfig index = 1;
+}
+
+/// Configuration of the entities index
+message EntityIndexConfig {
+    uint64 chain_index_min_depth = 1;
+
+    uint64 chain_index_depth_leeway = 2;
+
+    MutationIndexConfig pending_index_config = 3;
```

Comment:
I'd drop the `_config` suffix since the context is clear that we're expecting a config here

Reply:
k